src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -224,21 +236,23 @@ class CrossDeviceLinkUI extends Component {
             { invalidNumber && <div className={style.numberError}>{translate('errors.invalid_number.message')}</div> }
           </div>
           <div className={style.copyLinkSection}>
-            <div className={`${style.label}`}>{translate('cross_device.link.copy_link_label')}</div>
-              <div className={classNames(style.linkContainer, {[style.copySuccess]: this.state.copySuccess})}>
-                <textarea className={style.linkText} value={mobileUrl} ref={(element) => this.linkText = element}/>
-                { document.queryCommandSupported('copy') &&
-                  <div className={style.actionContainer}>
-                    <button
-                      type="button"
-                      onClick={this.copyToClipboard}
-                      className={style.copyToClipboard}
-                    >
-                      {linkCopy}
-                    </button>
-                  </div>
-                }
-              </div>
+            <div tabIndex="0" className={`${style.label}`}>{translate('cross_device.link.copy_link_label')}</div>
+            <div className={classNames(style.linkContainer, {[style.copySuccess]: this.state.copySuccess})}>
```

Comment:
[Minor] For simple conditions for dynamic styles you can just do: `classNames(style.linkContainer, his.state.copySuccess && style.copySuccess)` - IMO it's shorter.

Reply:
Ah, cool.